apm-sniffer/optional-plugins/quartz-scheduler-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/quartz/JobRunShellMethodInterceptor.java

```diff
@@ -48,14 +49,15 @@
     public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes, MethodInterceptResult result) throws Throwable {
         JobDetail jobDetail = (JobDetail) objInst.getSkyWalkingDynamicField();
 
-        String jobName = jobDetail.getKey().getName();
         String jobGroup = jobDetail.getKey().getGroup();
-        String operationName = ComponentsDefine.QUARTZ_SCHEDULER.getName() + "/" + jobName;
+        String jobName = jobDetail.getKey().getName();
+        String operationName = ComponentsDefine.QUARTZ_SCHEDULER.getName() + "/" + jobDetail.getJobClass().getName();
```

Comment:
This change related to the agent readme doc. Please fix.

Reply:
Please check again